File: CS50-6-Python/hello/hello.py
# The 'hello' program gets user input (a name), and will print out hello and the user's name
# It shows different ways to print strings with variables
# Note: get_string is specific to the cs50 library

# Add on variable using the + (adds without space)
answer = input("What is your name? ")
print("hello, " + answer)

# Add on variable using a comma (adds with space)
answer = input("What is your name? ")
print("hello,", answer)

# Use f format code within the brackets to substitute variables in the curly braces
answer = input("What is your name? ")
print(f"hello, {answer}")

File: CS50-1-C/hello/hello.c
// The "hello" program allows the user to input their first and last name, and prints out saying hello to the user with their first and last name

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get the user to input their first and last name
    string first = get_string("What is your first name?: ");
    string last = get_string("What is your last name?: ");

    // Prints hello with the first and last name
    printf("Hello, %s %s! \n", first, last);
}
